README.md

```diff
@@ -48,7 +48,7 @@ Provides common code quality guidelines.
 Microsoft.CodeQuality.Analyzers.Exp
 --------------------------------
 
-*Latest version:* Not Yet Available
+*Latest pre-release version:* [![NuGet](https://img.shields.io/myget/v/Microsoft.CodeQuality.Analyzers.Exp.svg)](https://dotnet.myget.org/feed/roslyn-analyzers/package/nuget/Microsoft.CodeQuality.Analyzers.Exp)
```

Comment:
One of the servers is running slowly right now so I can't verify, but I believe the image here should be this:

```
![MyGet](https://img.shields.io/dotnet.myget/roslyn-analyzers/v/Microsoft.CodeQuality.Analyzers.Exp.svg)
```

Reply:
That package will be deleted from master soon. DFA rules will merge into existing packages in features/dataflow branch.